packages/node/src/index.ts

```diff
@@ -78,7 +78,6 @@ export { defaultIntegrations, init, defaultStackParser, getSentryRelease } from
 export { addRequestDataToEvent, DEFAULT_USER_INCLUDES, extractRequestData } from '@sentry/utils';
 export { deepReadDirSync } from './utils';
 export { getModuleFromFilename } from './module';
-export { enableAnrDetection } from './anr';
```

Comment:
Removing this export is a breaking change, and we released the feature in beta.

Can we figure out a way to make this backwards compatible?

Reply:
Yep, will add that back in!

Reply:
I've re-added this back in and included a test with the old api